presto-spi/src/main/java/com/facebook/presto/spi/plan/AggregationNode.java

```diff
@@ -394,6 +411,18 @@ public Aggregation(
             this.mask = requireNonNull(mask, "mask is null");
         }
 
+        public static AggregationNode.Aggregation removeDistinct(AggregationNode.Aggregation aggregation)
```

Comment:
This change doesn't seem to be used in this commit. Should it belong to another commit?

Reply:
Yes, moved this to 3rd commit.